examples/v2.1.0/helpers.json

```diff
@@ -61,11 +59,10 @@
             "request": {
                 "auth": {
                     "type": "basic",
-                    "basic": {
-                        "username": "abhijit",
-                        "password": "kane",
-                        "showPassword": false
-                    }
+                    "basic": [
+                        { "key": "username", "value": "abhijit" },
+                        { "key": "password", "value": "kane" }
+                    ]
```

Comment:
@kamalaknn - this `showPassword` will stay as non-serialisable `_postman_*` properties, in case it needs to be transported. But for all practical purposes, we do not need it in SDK.

Reply:
Makes sens 👍

Reply:
Yes, there's a pending task for dealing with the same.